IAMOTZ/ievents
# #153888227 API endpoint for adding more admin

#### What does this PR do?
It adds an API endpoint for adding more admin
#### Description of Task to be completed?
User should be able to add more admin
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- Create a `.env` file to produce all the environment variables needed
- run npm run server:dev to start the server
- using the super admin token, send a post request to api/v1/users/amdin with the email of a user that is already signed up 
- the user should be able to access the admin related features
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#153888227 
#### Screenshots (if appropriate)
None
#### Questions:
None
